### ISSUE TITLE
Install failure on Fedora 24, 25, 26, and 27

### ISSUE THREAD

**Issue by rickhg12hs** [JuliaMath/GSL.jl#59]
Here's an install attempt:

``` julia
$ SUDO_ASKPASS=/usr/libexec/openssh/gnome-ssh-askpass ./julia
               _
   _       _ _(_)_     |  A fresh approach to technical computing
  (_)     | (_) (_)    |  Documentation: http://docs.julialang.org
   _ _   _| |_  __ _   |  Type "?help" for help.
  | | | | | | |/ _` |  |
  | | |_| | | | (_| |  |  Version 0.5.0 (2016-09-19 18:14 UTC)
 _/ |\__'_|_|_|\__'_|  |  
|__/                   |  x86_64-redhat-linux

julia> Pkg.build("GSL")
INFO: Building GSL
Installing dependency gsl-devel via `sudo yum install gsl-devel`:
Redirecting to '/usr/bin/dnf install gsl-devel' (see 'man yum2dnf')

Last metadata expiration check: 0:02:21 ago on Mon Oct 24 16:15:24 2016.
Package gsl-devel-2.1-4.fc24.x86_64 is already installed, skipping.
Dependencies resolved.
Nothing to do.
Complete!
=================================[ ERROR: GSL ]=================================

LoadError: Provider BinDeps.PackageManager failed to satisfy dependency libgsl
while loading /home/rick/.julia/v0.5/GSL/deps/build.jl, in expression starting on line 30

================================================================================

================================[ BUILD ERRORS ]================================

WARNING: GSL had build errors.

 - packages with build errors remain installed in /home/rick/.julia/v0.5
 - build the package(s) and all dependencies with `Pkg.build("GSL")`
 - build a single package by running its `deps/build.jl` script

================================================================================


```

**Comment by rickhg12hs** [comment]
Same problem on Fedora 26.

`gsl-devel` with `libgsl.so` is already installed.  Why doesn't the install work?

**Comment by patrikreiske** [comment]
Same problem on Fedora 27.

```
julia> versioninfo()
Julia Version 0.6.0
Commit 903644385b* (2017-06-19 13:05 UTC)
Platform Info:
  OS: Linux (x86_64-redhat-linux)
  CPU: Intel(R) Core(TM) i5-3320M CPU @ 2.60GHz
  WORD_SIZE: 64
  BLAS: libopenblas (DYNAMIC_ARCH Sandybridge)
  LAPACK: libopenblasp.so.0
  LIBM: libopenlibm
  LLVM: libLLVM-3.9.1 (ORCJIT, ivybridge)

julia> Pkg.build("GSL")
INFO: Building GSL
Paket gsl-devel-2.4-3.fc27.x86_64 ist bereits installiert, wird übersprungen.
==========================================[ ERROR: GSL ]===========================================

LoadError: Provider BinDeps.PackageManager failed to satisfy dependency libgsl
while loading /home/patrik/.julia/v0.6/GSL/deps/build.jl, in expression starting on line 30

===================================================================================================

=========================================[ BUILD ERRORS ]==========================================

WARNING: GSL had build errors.

 - packages with build errors remain installed in /home/patrik/.julia/v0.6
 - build the package(s) and all dependencies with `Pkg.build("GSL")`
 - build a single package by running its `deps/build.jl` script

===================================================================================================
```

```
[patrik@T430 ~]$ dnf list installed gsl*
Installed Packages
gsl.x86_64                                      2.4-3.fc27                                @anaconda
gsl-devel.x86_64                                2.4-3.fc27                                @fedora  
[patrik@T430 ~]$ find /usr/lib64/ -name libgsl.*
/usr/lib64/libgsl.so.23
/usr/lib64/libgsl.so.23.0.0
/usr/lib64/libgsl.so
```

**Comment by simonbyrne** [comment]
Should now be fixed by #77/#78.
